src/ProjectOrigin.WalletSystem.Server/Services/GRPC/RegistryService.cs

```diff
@@ -6,7 +6,7 @@
 using ProjectOrigin.WalletSystem.Server.Options;
 using ProjectOrigin.WalletSystem.Server.Projections;
 
-namespace ProjectOrigin.WalletSystem.Server.Services;
+namespace ProjectOrigin.WalletSystem.Server.Services.GRPC;
```

Comment:
This is not an actual gRPC service, more an internal service that helps contact the registry, should probably be moved back to services... or elsewhere.